book/website/communication/citable/citable-linking.md

```diff
@@ -0,0 +1,64 @@
+(cm-citable-linking)=
+
+# Linking Research Objects
+
+Linking your data, code and article will ensure that you receive **credit** for your work and it will increase the visibility of your research outputs.
+There are several different ways to do this. 
+You can decide to 1) open up your data publicly, at the same time as for example the **preprint** of your article, or 2) keep the dataset closed until the article is published.
+
+## 1. Open the Research Objects
+
+The easiest way to connect your dataset to your publication is to publish the dataset first (for example, at the same time as the **preprint**). 
+This will provide you with the correct information for your **Data Availability Statement** and your references (see {ref}`Citing Data<cm-citable-cite-data>`), and will allow reviewers to include the dataset in their review. 
+If any adjustments are needed based on the review process you can often upload a new version of the dataset (see for example [Zenodo](https://zenodo.org/), [Figshare](https://figshare.com/), [4TU.ResearchData](https://data.4tu.nl/info//en/)). 
```

Comment:
- Suggestion 1: Rather than linking to the three example services, maybe we can link to https://the-turing-way.netlify.app/communication/citable/citable-steps.html#servers-that-provide-dois instead?
- Suggestion 2: Rephrase slightly:
    > If any adjustments are needed based on the review process you can often upload a new version of the dataset and get a new DOI (see [link to above section] for how to do so; some services allow you to have a canonical DOI that will always point to the latest version of the dataset).

Reply:
Rephrased and referred to the other section. I kept the examples in there though because not all the examples in that section provide clear versioning systems I think!